jira-dependency-graph.py

```diff
@@ -15,6 +16,7 @@
 # is only about 5 lines.
 
 GOOGLE_CHART_URL = 'http://chart.apis.google.com/chart?'
+MAX_SUMMARY_LENGTH = 30
```

Comment:
PS: it could be a good idea to make this a command-line argument in the future. It just prevents having extremely long nodes.